foundation/foundation-mda/src/test/resources/specifications/data-records-generation.feature

```diff
@@ -214,21 +214,26 @@ Feature: Generating pipeline resources
     Then the "PySparkPipeline" pipeline POM has a dependency on "example-data-records-python"
 
   @pipeline-generation @code-generation
-  Scenario: Pipeline pyproject includes data module by default
+  Scenario: Pipeline generates file store environment variables necessary for accessing a single configured file store
     Given a project named "example"
-    And a "data-flow" pipeline using "data-delivery-pyspark"
-    And a dictionary and 0 or more record models
-    When the profile "data-delivery-pyspark-pipeline" is generated
-    Then the pyproject.toml file has a dependency on "example-data-records"
```

Comment:
Q: Do we want to get rid of these tests?
```
    Given a project named "example"
    And a "data-flow" pipeline using "data-delivery-pyspark"
    And a dictionary and 0 or more record models
    When the profile "data-delivery-pyspark-pipeline" is generated
    Then the pyproject.toml file has a dependency on "example-data-records"
```

Reply:
**_Strongly_** suggest we introduce a new feature file for the Spark application values generation and leave this one untouched.

Reply:
Refactored to extract spark application specs into it's own feature files.